src/common/commonutils/DeviceInfoUtils.c

```diff
@@ -48,6 +48,24 @@ void RemovePrefixUpTo(char* target, char marker)
     }
 }
 
+void RemovePrefixUpToString(char* target, const char* marker)
+{
+    if ((NULL == target) || (NULL == marker))
+    {
+        return;
+    }
+
+    int targetLength = (int)strlen(target);

```

Comment:
nit: Seems like this assignment can be removed (just be set to zero). It is reassigned again below (line 63) before first usage.

Reply:
Well spotted.